src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/mobile_command_factory.cc

```diff
@@ -523,10 +528,14 @@ bool MobileCommandFactory::IsAbleToProcess(
     const int32_t function_id,
     const application_manager::commands::Command::CommandSource message_source)
     const {
+  LOG4CXX_DEBUG(logger_,
```

Comment:
@IGapchuk not sure that this is the best place for logging, this message will spam the SDL log

Reply:
Replace with trace and do not duplicate MobileCommandFactory::IsAbleToProcess:
It will present in logger string any way